gf-webapp/src/component/pageBorder/footer/footerLink/FooterLink.tsx

```diff
@@ -0,0 +1,37 @@
+import {NavLink} from "react-router-dom";
+import styles from "src/component/pageBorder/footer/Footer.module.scss";
+
+/**
+ * Footer NavLink props
+ */
+interface FooterLinkProps {
+  /**
+   * NavLink path (route, where to move)
+   */
+  to: string;
+  /**
+   * NavLink value (text)
+   */
+  value: string;
+}
+
+/**
+ * Footer navLink component
+ */
+export const FooterLink: React.FC<FooterLinkProps> = (props: FooterLinkProps) => {
+
+  const footerLinkHandler = (isActive: boolean) => {
+    return isActive ? styles.footer_link_active : styles.footer_link;
+  };
+
+
+  return (
+    <NavLink
```

Comment:
Leave only one empty line between the 25-28 lines

Reply:
Done